package shared.exceptions;

public class NameAlreadyInUseException extends UserException {

}
